openshift-install-powervs

```diff
@@ -1305,12 +1298,24 @@ function main {
       IBM_CLI_OS="osx"
       OC_CLI_OS="mac"
       ARCHIVE_TYPE="tar.gz"
+      VERSION=$(sw_vers -productVersion 2>/dev/null || echo "")
```

Comment:
Would be good to change the variable to OS_VERSION or PLATFORM_VERSION ?

Reply:
Yes changing this to `OS_VERSION`.

Reply:
Done.